network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_author_detail_header.html

```diff
@@ -0,0 +1,49 @@
+{% load l10n i18n localization static blog_tags wagtailimages_tags wagtailroutablepage_tags wagtailcore_tags %}
+
+{% get_root_or_page as blog_index_page %}
+
+<div class="tw-grid tw-grid-cols-12 large:tw-gap-40">
+    <div class="tw-col-span-12 large:tw-col-span-8">
+        <div class="tw-text-sm tw-text-gray-40">
+            <a class="tw-text-gray-40" href="{% pageurl blog_index_page %}">Blog</a>
```

Comment:
Here, I don't think we need `blog_index_page`. We can use `page` directly, as it will get the url for the `BlogIndexPage`:

```html
 href="{% pageurl page %}"
```

If we needed any of the routes defined in the `BlogIndexPage`, we could use the `wagtailroutablepage_tags`, e.g.:

```html
{% load wagtailroutablepage_tags %}

 href="{% routablepageurl page "authors_index" %}"
```